### ISSUE TITLE
wrong with running faster_rcnn_inception_v2 model(tensorflow) configured OpenVINO Backend.

### ISSUE THREAD

**Issue by wanglan0804** [opencv/opencv#11983]
System information (version)
OpenCV => 4.0
Operating System / Platform => Windows 10 64 Bit
Compiler => Visual Studio 2015
--CPU => Intel i7-6700
--GPU => intel HD graphics 530
Detailed description
The file "frozen_inference_graph.pbtxt" is same as the opencv_extra github's file "faster_rcnn_inception_v2_coco_2018_01_28.pbtxt".
when I used the code like this:
String model = "G:\faster_rcnn_inception_v2\frozen_inference_graph.pb";
String pbtxt = "G:\faster_rcnn_inception_v2\frozen_inference_graph.pbtxt";
dnn::Net net = cv::dnn::readNetFromTensorflow(model, pbtxt);
......
net.setPreferableBackend(DNN_BACKEND_OPENCV);
net.setPreferableTarget(DNN_TARGET_OPENCL);
......
everything is ok,the car and person in picture is regconized correctly.I want more performance in this application,but when I change the code using OpenVINO(net.setPreferableBackend(DNN_TARGET_INF_ENGINE),I get the wrong message:
OpenCV(4.0.0-pre) Error: Assertion failed (Failed to initialize Inference Engine backend: Invalid softmax axis -1) in cv::dnn::InfEngineBackendNet::initPlugin, file D:\opencv\modules\dnn\src\op_inf_engine.cpp, line 416.
Steps to reproduce
The sample code is changed according to opencv\samples\dnn\object_detection.cpp. The source code can be downloadde from web:https://github.com/opencv/opencv/issues/11961

**Comment by wanglan0804** [comment]
OK,thanks a lot,I'll be patient to wait the new version to get this feature.

**Comment by dkurt** [comment]
@wanglan0804, we've merged a PR with Faster-RCNN network using Intel's Inference Engine backend.
